build/Dependencies.props

```diff
@@ -51,10 +51,10 @@
   <!-- Test-only Dependencies -->
   <PropertyGroup>
     <BenchmarkDotNetVersion>0.12.0</BenchmarkDotNetVersion>
-    <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20080.1</MicrosoftCodeAnalysisTestingVersion>
+    <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20156.1</MicrosoftCodeAnalysisTestingVersion>
     <MicrosoftExtensionsTestPackageVersion>3.0.1</MicrosoftExtensionsTestPackageVersion>
     <MicrosoftMLTestDatabasesPackageVersion>0.0.6-test</MicrosoftMLTestDatabasesPackageVersion>
-    <MicrosoftMLTestModelsPackageVersion>0.0.6-test</MicrosoftMLTestModelsPackageVersion>
+    <MicrosoftMLTestModelsPackageVersion>0.0.7-test</MicrosoftMLTestModelsPackageVersion>
```

Comment:
Odd that we're making this change just now.  It looks like the change in testdata the reved this was in Feb 2020: https://github.com/dotnet/machinelearning-testdata/pull/22

Presumably with these changes:
https://github.com/dotnet/machinelearning-testdata/pull/21

I wonder if we need to make any changes to test cases to react to the model changes?

Reply:
Locally the build passes, so I am thinking no changes are needed. I think the version update is just because of the feed change. I want to run the full CI process though just to make sure, but I'm running into some issues kicking it off for some reason.

Reply:
Well, that's just a symptom of one version being on the feeds but the other not.

I went back and examined history some more and I actually think the 0.0.6->0.0.7 change was accidental for this package.  
https://github.com/dotnet/machinelearning-testdata/pull/21 changed the Microsoft.ML.Onnx.TestModels package, but then the Microsoft.ML.TestModels package was rev'ed in the subsequent PR. 

Later the Microsoft.ML.Onnx.TestModels version was rev'ed in
https://github.com/dotnet/machinelearning-testdata/pull/27
Likely because folks realized this was still missing the changes.

There's actually been no change to Microsoft.ML.TestModels content since 10/2019, so these revisions have the same content. https://github.com/dotnet/machinelearning-testdata/commits/master/Microsoft.ML.TestModels